spec/controllers/bookings_controller_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "rails_helper"
+
+RSpec.describe "Bookings", type: :request do
+  describe "GET /index" do
+    pending "add some examples (or delete) #{__FILE__}"
+  end
+end
```

Comment:
I asked you before but how about trying to make a few tests for your controller ? 🙂 
You can take inspiration from the EventsController spec, you don't have a lot of actions either so it shouldn't be too long